family2family/build.gradle

```diff
@@ -26,7 +30,7 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
     compile("org.springframework.boot:spring-boot-starter-data-rest")
-    compile("org.springframework.security.oauth:spring-security-oauth2:${springOauthVersion}")
+    compile("org.springframework.security.oauth:spring-security-oauth2")
```

Comment:
We won't get automatic upgrades...it's the version associated with the Spring Boot version.  See https://github.com/spring-projects/spring-boot/blob/v1.5.2.RELEASE/spring-boot-dependencies/pom.xml